core/safe_refcount.h

```diff
@@ -40,6 +40,22 @@ uint32_t atomic_conditional_increment(register uint32_t *counter);
 uint32_t atomic_decrement(register uint32_t *pw);
 uint32_t atomic_increment(register uint32_t *pw);
 
+#ifdef NO_THREADS
+
```

Comment:
please add these in the .cpp like the other functions above, so we avoid namespace pollution with the windows.h include